package.json

```diff
@@ -8,8 +8,8 @@
     "test-ci": "lerna run test-ci --stream --no-private",
     "watch": "lerna run watch --parallel",
     "build": "lerna run build --parallel --no-private",
-    "format": "packages/node_modules/eslint/bin/eslint.js --fix \"packages/*/src/**/*.js\"",
-    "lint": "packages/node_modules/eslint/bin/eslint.js \"packages/*/src/**/*.js\"",
+    "format": "node_modules/eslint/bin/eslint.js --fix \"packages/*/src/**/*.ts\"",
+    "lint": "node_modules/eslint/bin/eslint.js \"packages/*/src/**/*.ts\"",
```

Comment:
yeh same thoughts. Leaving this for later when start addressing the react modules. We should look into using the `ext` parameter instead of the extension being part of the pattern.

Reply:
Wow, didn't know about `ext` parameter, seems super useful here.